functions/Backup-DbaDatabase.ps1

```diff
@@ -71,27 +75,47 @@ Backs up AdventureWorks2014 to sql2016's C:\temp folder
 #>
 	[CmdletBinding(DefaultParameterSetName = "Default")]
 	param (
-		[parameter(ParameterSetName = "NoPipe", Mandatory = $true)]
+		[parameter(ParameterSetName = "NoPipe", Mandatory = $true)][parameter(ParameterSetName = "Pipe")]
 		[object[]]$SqlInstance,
+		[parameter(ParameterSetName = "NoPipe")][parameter(ParameterSetName = "Pipe")]
 		[System.Management.Automation.PSCredential]$SqlCredential,
-		[string]$BackupDirectory,
+		[parameter(ParameterSetName = "NoPipe")][parameter(ParameterSetName = "Pipe")]
+		[string[]]$BackupDirectory,
+		[parameter(ParameterSetName = "NoPipe")][parameter(ParameterSetName = "Pipe")]
 		[string]$BackupFileName,
+		[parameter(ParameterSetName = "NoPipe")][parameter(ParameterSetName = "Pipe")]
 		[switch]$NoCopyOnly,
+		[parameter(ParameterSetName = "NoPipe")][parameter(ParameterSetName = "Pipe")]
 		[ValidateSet('Full', 'Log', 'Differential', 'Diff', 'Database')]
 		[string]$Type = "Database",
-		[parameter(ParameterSetName = "Pipe", Mandatory = $true, ValueFromPipeline = $true)]
-		[object[]]$DatabaseCollection
+		[parameter(ParameterSetName = "Pipe",Mandatory = $true, ValueFromPipeline = $true)]
+		[object[]]$Arrdatabases,
```

Comment:
I'll fix the help

Yes, name could be better. I think I went with something different as the others get clobbered by the  dynamic parameters bit. but now that's escaped it might not be an issue